.github/workflows/test-node.yml

```diff
@@ -45,6 +51,39 @@ jobs:
 
       - run: ${{ (inputs.test_type == 'node-optional') && 'npm run test:optional --workspace=@waku/tests' || 'npm run test:node' }}
 
+      - name: Merge allure reports
+        if: always() && env.ALLURE_REPORTS == 'true'
+        run: node ci/mergeAllureResults.cjs 
+
+      - name: Get allure history
```

Comment:
Since reports are in the separate repo now - do you think it will slow down if we will continue doing so for long period of time? 

Also, as I understand we keep history from the test run in `nwaku`, do we differentiate between tests run in `js-waku` or `nwaku`?